### ISSUE TITLE
PinballFX3: They could do the same way as OpenBor

### ISSUE THREAD

**Issue by fabyo** [fabricecaruso/batocera-ports#150]
Placing the tables (.pxp) in the roms/pinballfx3 folder and when running the table, copy it to the data\\steam folder
it would be in the default of OpenBor, and leaving the tables in the roms folder without having to create bats, etc.

            if (Directory.Exists(pathPXP))
            {
                destFile = Path.Combine(pathPXP, Path.GetFileName(rom));
                if (!File.Exists(destFile))
                {
                    File.Copy(rom, destFile);
                }
            }
...
        public override void Cleanup()
        {
            if (destFile != null && File.Exists(destFile))
            {
                File.Delete(destFile);
            }

            base.Cleanup();
        }  

The idea is to avoid using .bat files and keep everything in the same standard.

**Comment by Tartifless** [comment]
Kind of a stretch this one.
Currently you can just put an empty txt file and just rename it as your table name, no need to copy the actual table in the roms folder.
In the end it is still the table from the steam folder that is played.

Moreover for people wanting to play directly from Steam, tables should remain in the steam folder.

**Comment by fabyo** [comment]
Yes, in my case I used "pinballfx-hack" to copy and delete only if that's the core

thanks

**Comment by fabyo** [comment]
another solution that I've used, just commenting for knowledge is the use of the "mklink" command of windows, where it mirrors the same folder pointing the roms to the folder where they are, it's also very useful, organized but few people like it that way, but thanks for the feedback

**Comment by fabricecaruso** [comment]
Why dont you submit PRs ?

**Comment by fabyo** [comment]
When I forked the project, my Visual Studio automatically updated the entire project to framework 4.8.1, and I started making many unnecessary changes. I thought it wouldn't be compatible to submit a PR, so I even stopped updating my GitHub repository. However, I've created a new development environment with the correct framework version, and now it's compatible. I haven't uploaded it to my GitHub yet, but I'm going to reset everything and start over following your standards, so I can try to contribute more to this excellent project.

thanks.
